pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -69,6 +70,11 @@ type CatalogSourceSpec struct {
 	// +Optional
 	Image string `json:"image,omitempty"`
 
+	// GrpcPodConfig exposes different overrides for the pod spec of the CatalogSource Pod.
+	// Only used when SourceType = SourceTypeGrpc and Image is set.
+	// +Optional
```

Comment:
Is the `+Optional` flag case insensitive? I've only ever seen it written as `+optional`.

Reply:
That's a great question. I just kept what was around. I've updated everything to be lowercase to keep it with the general lower-case convention.